R/create_hexsticker.R

```diff
@@ -0,0 +1,29 @@
+#' @title Make hexagonal logo for package
+#'
+#' @description This function makes a hexsticker in INBO style for the provided
+#' package name.
+#'
+#' @param package_name package name that should be mentioned on the hexsticker
+#' @param filename filename to save the sticker
+#'
+#' @return A figure is saved in the working directory or provided path
+#'
+#' @export
+#'
+#' @importFrom hexSticker sticker
+#'
+#' @family utils
+#'
+#' @examples
+#' create_hexsticker("checklist", filename = "hexsticker.png")
+
+create_hexsticker <- function(package_name, filename = "hexsticker.png") {
+  background <- system.file("inbo-empty.png", package = "checklist")
+  sticker(
+    background, s_x = 1, s_y = 1, s_width = 1, s_height = 1, asp = 0.85,
+    package = package_name, p_y = 0.75, p_color = "#000000",
+    p_family = "Flanders Art Sans", p_size = 24, p_fontface = "bold",
```

Comment:
The function could not find the font on my machine. Although it is installed.

Reply:
Hmm, I'm not sure how to solve this.  Apparently I should load the font first (and then use `showtext` to make it available for the hexsticker), but after installing it, it does not show up in the list generated by `extrafont::fonts()` (in my case).  Do you know for any chance how I can activate the font in R?

Reply:
Ah, found it, they are added after running `extrafont::font_import()`!

Reply:
The committed code works on my machine, but it will fail (give an error) if the font is not installed.  Is this ok, or do you prefer the previously given warning and using another font?

Reply:
I eliminated this error, but the example still gives an error because it apparently has no access to the installed package hexSticker.

Reply:
you forgot to list `extrafont` under `Suggests` in the DESCRIPTION

Reply:
ok, added the font to the package and removed the check for presence of the font, by which `extrafont` is not used anymore

Reply:
I've tested is on my machine and it works fine.